linux.md

```diff
@@ -42,3 +42,10 @@ tar -zcvf foo.txt.tar.gz foo.txt
 tar -xvf foo.txt.tar.gz
 
 ```
+## Text Editor Options for Programmers
```

Comment:
We don't think this information helps in cheatsheet, instead, you can add commands for vm/ nano